### DISCUSSION TITLE
Unable to upload a file in INSPIRE validator

### DISCUSSION THREAD

**Discussion by aantgis** [INSPIRE-MIF/helpdesk-validator#1004] [Ideas]
Before reporting an issue with the INSPIRE Reference Validator deployed at JRC, please:
- check whether a similar issue has already been opened
- try to validate your resource after cleaning the browser cache (in case of error during the upload of the resource) 
- check the [list of known limitations](https://github.com/INSPIRE-MIF/helpdesk-validator/wiki/Known-limitations).

To report an issue, please replace these lines with a brief summary of your issue and the expected behaviour.

To help us investigate the issue, you may want to include:

- a description of the steps to reproduce your issue
- the browser and version used
- a screenshot illustrating the problem
- the Test Report (ideally as an attachment to the issue)
- the name of the Test that failed or better: change the `Level of Detail`
in the Report by clicking on `All Details`, then scroll down to the failed test
and copy the Link on the right-hand side of the `Assertion URI`
- the URL of your service / the file you have uploaded or referenced (as
an attachment).

If your service is protected or you have confidential data that you cannot
upload here, you can request an email address to which you can send URLs / files.
Please note that issues cannot be investigated for services/datasets that we cannot
access.

**Comment by aantgis** [comment]
I am trying for several days to test an xml file but while trying to upload I receive the following error message:

ERROR
An error has occured uploading file (Code: 400 - )

in a popup message at the bottom right side of the screen.
I have also tried to upload a zip file and even services but with no luck.
Please advise 
Anthony Antoniou

**Comment by fabiovinci** [comment]
Dear @aantgis,

could you please provide your resource for internal testing?

**Comment by aantgis** [comment]
please find attached the zip xml

[addresses_location.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/12050081/addresses_location.zip)

**Comment by fabiovinci** [comment]
Dear @aantgis,

the validator is not able to load the XML file since it does not recognize the resource type.

Your XML file is not encoded according to the [INSPIRE Addressed application schema](https://inspire.ec.europa.eu/schemas/ad/4.0/Addresses.xsd), see [here](https://b5m.gipuzkoa.eus/inspire/wfs/gipuzkoa_wfs_ad?request=GetFeature&Language=eng&CRS=urn:ogc:def:crs:EPSG::4326&DataSetIdNamespace=Gipuzkoa_Provincial_Council&service=WFS&count=2&STOREDQUERY_ID=http://inspire.ec.europa.eu/operation/download/GetSpatialDataSet&version=2.0.0&DataSetIdCode=ES.GFA.AD.MD) an example of how it should be encoded.

**Comment by aantgis** [comment]
Thank you for the clarification.
I thought that the validator was supposed to provide these error messages
and warnings!

**Comment by fabiovinci** [comment]
**Possible enhancement:** The Validator should provide a more specific error message when it is not able to recognize the resource type.
